dmscripts/generate_questions_csv.py

```diff
@@ -45,11 +47,31 @@ def return_rows_for_sections(sections):
 
     for section in sections:
         for question in get_questions(section.questions):
+            # Using get_source because we want the original markdown, not the rendered HTML
+            # this means that no context can be reflected in the question content, i.e. our passed-in
+            # context can only influence whether a question is displayed or not.
+            # TODO if this turned out to matter, instead of get_source, we could convert back to markdown(!)
+
+            parent = question.get('parent')
+            page_title = parent.get_source('question') if parent else None
+            section_and_page_title = ' / '.join(filter(None, [section.name, page_title]))
+
+            # section.description is no longer expected to be used
```

Comment:
When you say "no longer expected to be used" - you mean just for the purposes of this CSV file?